### PR TITLE
Origin/release note v3

### PR DESCRIPTION
**Pull request by petra-syskit**

### CODE REVIEW

File: product-updates/security-manager-3-release-note.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+author: Petra Filipi 
+date: 18/07/2018 
+description: This article describes what's new and improved in the latest version of SysKit Security Manager.
+---
+
+## We're excited to present SysKit Security Manager 3.0
+
+SysKit Security Manager is centralized __SharePoint and Office 365 security reporting and management tool__. It was created to help SharePoint global and site collection administrators with controlling the security in their SharePoint and Office 365 environment.
+
+Tool has gone through some changes since the last version. One of those changes is that entering password for SharePoint Online directy into SysKit Security Manager is no longer required. From now on we are using the default sign in experience.
```
Comment (tosirovect2) [inline product-updates/security-manager-3-release-note.md]: directly*.
from now on**,**

File: faq/index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ This article lists some of the commonly asked questions.
 Yes, SysKit Security Manager works with both SharePoint Online and On-Premise site collections.
 
 ## Can I connect to multiple site collections at once? 
-Right now, connecting to multiple site collections at once is not supported. But it is something we want to add in future versions. If you want to connect to more than one site collection, you should repeat the connect process. 
+Yes. If you are a global tenant admin in Office 365 or a farm admin in SharePoint On-Premise, you will be able to automatically discover and connect to all site collections (including Office 365 Groups) in the tenant.
```
Comment (imesaric) [inline faq/index.md]: možda 'If you are a global admin in an Office 365 tenant or a farm admin in a SharePoint On-Premise environment..',>
global tenant admin mi ne paše baš

File: product-updates/security-manager-3-release-note.md (hunk 1 above)
Comment (imesaric) [inline product-updates/security-manager-3-release-note.md]: sign-in experience*

File: product-updates/security-manager-3-release-note.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+[Try SysKit Security Manager Free](https://www.syskit.com/products/security-manager/download/)
+
+__Product version:__ 3.0.0  
+__Build number:__      
+__Release date:__ July 18, 2018  
+
+
+## Features
+* __Automatic Discovery of Site Collections and Office 365 Groups for Global and Farm Administrators__ – If you are a global admin in Office 365 or a farm admin in SharePoint On-Premise, you will be able to automatically discover and connect to all site collections and Office 365 Groups in the tenant.
+* __Ability to Import Site Collection List from File__– If you are a site collection admin or a farm admin you can connect to multiple site collections at one time by importing site collection URLs from a text file.  Site collection URLs in file must be newline separated.
```
Comment (imesaric) [inline product-updates/security-manager-3-release-note.md]: at one time -> at the same time
site collection URLs in the file...

File: product-updates/security-manager-3-release-note.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+__Product version:__ 3.0.0  
+__Build number:__      
+__Release date:__ July 18, 2018  
+
+
+## Features
+* __Automatic Discovery of Site Collections and Office 365 Groups for Global and Farm Administrators__ – If you are a global admin in Office 365 or a farm admin in SharePoint On-Premise, you will be able to automatically discover and connect to all site collections and Office 365 Groups in the tenant.
+* __Ability to Import Site Collection List from File__– If you are a site collection admin or a farm admin you can connect to multiple site collections at one time by importing site collection URLs from a text file.  Site collection URLs in file must be newline separated. 
+* __Ability to Take Partial Snapshots__ – Take snapshot option makes a snapshot of your environment. Now you can choose which site collections will be included in snapshot. Decide either to auto discover and load new site collections and Office 365 groups or manage site collections and Office 365 groups for snapshot from existing connections.
```
Comment (imesaric) [inline product-updates/security-manager-3-release-note.md]: will be included in the* snapshot
auto discover -> auto-discover*
zadnja rečenica - nije mi najjasniji ovaj drugi dio, možda ovako: Decide either to auto-discover and load new site collections and Office 365 groups, or to load site collections and Office 365 groups from existing connections.

File: product-updates/security-manager-3-release-note.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+* __Security Audit Report__
+    * __Users with Privileged Access__ – Shows all users with privileged access on the tenant or farm, such as farm and site administrators, or primary and secondary administrators, Users with Full Control / Full Read Web Application Policy, etc.
+* __Health Checks Reports__
+    * __Directly Assigned Permissions__– Shows users who have directly assigned permissions to securable objects within the selected site collections. Avoid granting permissions directly to individuals and work with groups instead.
```
Comment (imesaric) [inline product-updates/security-manager-3-release-note.md]: ...assigned permissions on* securable objects...

**Pull request merged**